subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/TestPageGenerator.java

```diff
@@ -187,10 +189,10 @@ private PerformanceTestExecution findPreviousExecutionInSameBranch(PerformanceTe
             }
 
             private void addPerformanceGraphs() {
-                List<Chart> charts = Lists.newArrayList(new Chart("totalTime", "total time", "s", "totalTimeChart"));
+                List<Chart> charts = Lists.newArrayList(new Chart("totalTime", "total time", "s", "totalTimeChart", false));
                 if(testHistory instanceof CrossVersionPerformanceTestHistory) {
-                    charts.add(new Chart("confidence", "confidence", "%", "confidenceChart"));
-                    charts.add(new Chart("difference", "difference", "%", "differenceChart"));
+                    charts.add(new Chart("confidence", "confidence", "%", "confidenceChart", true));
+                    charts.add(new Chart("difference", "difference", "%", "differenceChart", false));
```

Comment:
I was expecting that we add the background to the difference graph and the sign to the confidence graph.